lib/rust/ensogl/core/src/display/object/instance.rs

```diff
@@ -939,80 +939,6 @@
 //! node2.allow_grow();
 //! ```
 //!
-//! ## The content origin
-//! In the parent container size is set to hug, the container size will be automatically increased
-//! to fit the children, however, the children will not be moved, so they can still overflow the
-//! parent container. In such a case, the content origin point is set to the minimum X-axis and
-//! Y-axis of the content. For example, the following code creates a manual layout with three that
-//! overflow the parent container:
-//! ```
-//! //     ╭──────── ▶ ◀ ──────────────────╮
-//! //     │ root                          │
-//! //     │                 ╭────────╮    │
-//! // ╭───┼────╮            │ node3  │    │
-//! // │ node1  │            │        │    ▼
-//! // │   │    │ ╭────────╮ ╰────────╯    ▲
-//! // ╰───┼────╯ │ node2  │               │
-//! //     ╰──────┼────────┼───────────────╯
-//! //   ╱        │        │           
```

Comment:
1. Docs should mention that the auto layout works only with bottom-left aligned things. This should be a big warning with a proper explanation of why old shapes will not work correctly. Also, I think that now, instead of having alignment as parameter to `ensogl::shape!` we should rename it to `ensogl::shape_old2` or something like that and keep `ensogl::shape!` always aligned to bottom left corner.
2. Could you please update docs with explicit info that the origin is always bottom left corner, please?
3. Could you add to docs example with an overflow of manual layout, please?

Reply:
I would like to avoid introducing another deprecated shape macro. Instead, I will change the default to bottom_left, and override it in all existing shapes to center.